src/server.c

```diff
@@ -564,6 +564,10 @@ struct redisCommand redisCommandTable[] = {
      "write fast @sortedset",
      0,NULL,1,1,1,0,0,0},
 
+    {"zmpop", zmpopCommand,-4,
+     "write @sortedset",
+     0,zmpopGetKeys,0,0,0,0,0,0},
```

Comment:
Wasn't the key spec change merged? I think this needs to be updated now.

Reply:
Yes. I am writing the test code. Should come out this weekend. After that i will catch up the unstable branch